cetmix_tower_server/models/cx_tower_file_template.py

```diff
@@ -179,5 +179,7 @@ def create_file(self, server, server_dir="", if_file_exists="raise"):
                 "source": self.source,
             }
         )
+        if existing_file and if_file_exists == "overwrite":
+            existing_file.unlink()
```

Comment:
This may [physically delete](https://github.com/cetmix/cetmix-tower/blob/c653f3bae1e144130c6a741cd5279e0b3b3a89c8/cetmix_tower_server/models/cx_tower_file.py#L299) file from server. Is this what we really want to do?